inst/tests/tests.Rraw

```diff
@@ -15727,7 +15727,8 @@ DT = data.table(z = 1i)
 test(2069.33, DT[DT, on = 'z'], error = "Type 'complex' is not supported for joining/merging")
 
 # forder verbose message when !isReallyReal Date, #1738
-DT = data.table(d=sample(seq(as.Date("2015-01-01"), as.Date("2015-01-05"), by="days"), 20, replace=TRUE))
+date_dbl = as.Date(as.double(seq(as.Date("2015-01-01"), as.Date("2015-01-05"), by="days")), origin="1970-01-01")
```

Comment:
do we still need to edit this test? I'd've thought one effect of this PR is to make the old test keep working as intended...

Reply:
Yes, we alter this test to become future proof and ensure that we have a indeed a `double`, no matter what `R` decides what the underlying type should be